forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -171,17 +173,15 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Captures a Forage Payment associated with an EBT card
-     *
-     * @param params The parameters required for payment capture, including
-     * reference to a ForagePINEditText and a Payment ref
+     * Captures a payment via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element.
      *
-     * @return A ForageAPIResponse indicating the success or failure of the
-     * payment capture. On success, returns a confirmation of the transaction.
-     * On failure, provides a detailed error response.
-     *
-     * @throws ForageConfigNotSetException If the passed ForagePANEditText instance
-     * hasn't had its ForageConfig set via .setForageConfig().
+     * @param CapturePaymentParams A model that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance and a `paymentRef`, returned by the [Create a Payment](https://docs.joinforage.app/reference/create-a-payment) endpoint, as the [CapturePaymentParams] that Forage uses to capture a payment. 
+     * @return A [ForageApiResponse] object. 
+     * 
+     * On success, the object confirms the transaction. 
+     * 
+     * On failure, the object includes a list of [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can unpack to troubleshoot the issue. 
+     * @throws [ForageConfigNotSetException] If the [ForageConfig] is not set for the provided `foragePinEditText`.
      */
     override suspend fun capturePayment(params: CapturePaymentParams): ForageApiResponse<String> {
```

Comment:
as above, maybe worth linking relevant capture payment error-handling pages:

- [SDK errors](https://docs.joinforage.app/reference/errors#sdk-errors)
- [Test EBT Cards for triggering Payment Capture exceptions](https://docs.joinforage.app/docs/test-ebt-cards#payment-capture-exceptions)